src/modules/languages/python.nix

```diff
@@ -135,6 +145,12 @@ in
       description = "Whether `pip install` should avoid outputting messages during devenv initialisation.";
     };
 
+    venv.patchelf = lib.mkOption {
```

Comment:
I wonder if this should be a no-op on non-elf architectures to not break macOS by accident.